### ISSUE TITLE
"Change file permissions" step reported as failed on OSX Mojave

### ISSUE THREAD

**Issue by RussKie** [dotnet/templating#1743]
## The environment:
OSX 10.14.1
```
~/MacDev$ /bin/bash --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin18)
Copyright (C) 2007 Free Software Foundation, Inc.

~/MacDev$ dotnet --version
2.1.302

~/MacDev$ dotnet new --debug:version
Template Instantiation Commands for .NET Core CLI

 Version:     1.0.2-beta3
 Commit Hash: c0f84b79c361e961a699a5c70d255db11c0d411c
```
My template.json (snip):
```
   "postActions": [
      {
         "actionId": "cb9a6cf3-4f5c-4860-b9d2-03a574959774",
         "condition": "(OS != \"Windows_NT\")",
         "description": "Make scripts executable",
         "manualInstructions": [
            {
               "text": "Run 'chmod +x *.sh'"
            }
         ],
         "args": {
            "+x": "*.sh"
         },
         "continueOnError": true
      },
```

## The issue
When I execute the template I get the following output:
```
~/MacDev$ dotnet new  mytemplate -n 'myprojectname' --force
The template "My Template" was created successfully.

Processing post-creation actions...
The files /Users/user/MacDev/myprojectname/chmod, /Users/user/MacDev/myprojectname/+x, and /Users/user/MacDev/myprojectname/*.sh donot exist.
Unable to apply permissions +x to "*.sh".
Post action failed.
Description: Make scripts executable
Manual instructions: Run 'chmod +x *.sh'
```

However it seems that permissions _do_ get applied, as the following steps in the template run those scripts without errors.

**Comment by donJoseLuis** [comment]
1. Adding item to specific project corresponding to a targeted delivery area
1. Once this has been done for the entire backlog, the next steps will be to prioritize within each project & output a delivery roadmap.
1. At that point, the issues will be assigned to a corresponding milestone
1. The team will then begin to deliver against the road map, contributions always welcome

**Comment by AR-May** [comment]
Hello @RussKie, thank you for the detailed information.
This issue should be fixed within PR #1555 and PR #2579, the root of the problem seems to be the same as in issue #2072.
